net/httpsniff/src/TRootSnifferFull.cxx

```diff
@@ -553,9 +553,11 @@ Bool_t TRootSnifferFull::ProduceExe(const std::string &path, const std::string &
          // very special case - function requires list of options after method=argument
 
          const char *pos = strstr(options.c_str(), "method=");
-         if (!pos || (strlen(pos) < strlen(method_name) + 8))
+         if (!pos || (strlen(pos) < strlen(method_name) + 7))
             return debug != nullptr;
-         call_args.Form("\"%s\"", pos + strlen(method_name) + 8);
+         const char *rest_url = pos + strlen(method_name) + 7;
+         if (*rest_url == '&') ++rest_url;
+         call_args.Form("\"%s\"", rest_url);
```

Comment:
Since the arguments comes from an arbitrary web query, shouldn't there be some sort of argument sanitization?

Reply:
It is kind of arguments forwarding to C++ method.
In principle -yes - one have to check for eventual quotes here, otherwise string may be inorrectly handled.

Reply:
But this PR fixes other problem, I can create another PR for fixing quotes.
In principal, I plan to add special tests (to rottest) to check many of such special cases in THttpServer